tests/usage.test.ts

```diff
@@ -28,6 +29,7 @@ const examplesToTest = [
   priorApprovalBuildHomes,
   priorApprovalConvertCommercialToHome,
   priorApprovalLargerExtension,
+  landDrainageConsent,
```

Comment:
Confusing not getting any type errors, but tests are failing validation on `data.proposal` !